apps/api-server/src/services/checkHostStatus.js

```diff
@@ -141,37 +141,40 @@ const checkHostStatus = async (conditions) => {
       //ensure it's an object so we dont have to worry about checks later
       hostStatus = hostStatus ? hostStatus : {};          //
 
-      const domainIp = getDomainIp(project.url);
+      const domainIp = await getDomainIp(project.url);
 
       hostStatus.ip = domainIp !== null && domainIp === serverIp ? true : false;
```

Comment:
Persoonlijk voor de leesbaarheid zou ik hostStatus.ip hernoemen naar hostStatus.hasIp of iets in die trant (ga er vanuit omdat het een boolean als waarde heeft).  Daarnaast kan 

`hostStatus.ip = domainIp !== null && domainIp === serverIp ? true : false;` naar `hostStatus.ip = domainIp !== null && domainIp === serverIp` gezet worden.

Reply:
Valide punt! Ik heb dit doorgevoerd, en gekozen voor `hostStatus.dnsRecordIsSetToCorrectIp` zodat nu duidelijk is waar die boolean eigenlijk voor staat.